python/templates/index.ejs

```diff
@@ -40,6 +41,11 @@
               <h3 class="item-overview__heading">access_token</h3>
               <p class="item-overview__id" id="access_token">••••••••hsakjsl</p>
             </div>
+            <!-- The payment_token is only relevant for the UK Payment Initiation product. -->
+            <div class="payment_initiation item-overview__column" style='display:none;'>
+              <h3 class="item-overview__heading">payment_token</h3>
+              <p class="item-overview__id" id="payment_token">••••••••hsakjsl</p>
```

Comment:
Should this be a different example token than the access token from above?

Reply:
The shape of both tokens is the same, and this value of the payment token only shows up when the `payment_initiation` product is used, in which case this value is substituted with the value of the generated payment token. So I believe this HTML should be good as is.

Note: Payment Token looks like `payment-token-sandbox-00a0000a-aaaa-0000-aa00-00000aaa00a0`. Access token looks like `access-sandbox-aa000000-00a0-0000-000a-a0000000a000`.